src/Signin.tsx

```diff
@@ -92,8 +99,20 @@ export class Signin extends PureComponent<Props, State> {
 
   private checkForLogin = async () => {
     try {
-      await client.users.me()
-
+      const user = await client.users.me()
```

Comment:
i think this goes here:
https://github.com/influxdata/ui/blob/d44fdb618b2a857ee45995d987af01d276158cef/src/me/actions/thunks/index.ts#L22